src/styles/_catalog.scss

```diff
@@ -1,10 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
 
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
```

Comment:
According to the task,  four columns start at 1024px, but you have 3 columns. The same problem with 768, 488, and less than 488 px
![image](https://user-images.githubusercontent.com/119349933/208301584-643edfbe-6dcc-481e-b991-b1592797727e.png)